etcdserver/api/v3rpc/key.go

```diff
@@ -39,6 +41,7 @@ type kvServer struct {
 }
 
 func NewKVServer(s *etcdserver.EtcdServer) pb.KVServer {
+	MaxOpsPerTxn = int(s.Cfg.MaxOpsPerTxn)
```

Comment:
this will break embedding when using multiple etcd servers with different max txn op configurations